react/config.js

```diff
@@ -0,0 +1,9 @@
+const config = {
+  // Set to "true" to showe the Day of the Week filter in the home page.
+  showDayOfTheWeekFilter: false,
+
+  // Set to "true" to showe the Product Type in the home page.
```

Comment:
The word "show" has a typo in both comments here.

Reply:
Thanks for the observation, corrected the typo.